### PR TITLE
Replaces reproj image.tests.js and tile.tests.js

### PR DESCRIPTION
**Pull request by KaiVolland**
Transforms the old rendering tests for `ReprojImage` and `ReprojTile` to the new rendering test approach.

### CODE REVIEW

**Comment by ahocevar** [comment]
@KaiVolland I could migrate one of the existing reproj tests so you see how it could be done. Would that help?

**Comment by KaiVolland** [comment]
@ahocevar Would be awesome.
Im still working on this but my progress is very slow and im just slowly beginning to understand what the class is doing.

**Comment by ahocevar** [comment]
@KaiVolland Take a look at 789767d4119f8e25e284a1b4f8ba6e0d946bd0cf, and feel free to cherry-pick that into your pull request.

**Comment by KaiVolland** [comment]
@ahocevar The class `ReprojImage` is not used in your referenced commit.
I understand, that the `ReprojImage` is used internally. So is it enough to just add a layer with a different projection as the one of the map to test the rendering?

**Comment by KaiVolland** [comment]
Let me know if this suits the needs to test `ReprojImage` and `ReprojTile`. If yes, I'll clean up the commithistory.

File: rendering/cases/reproj-image/main.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import Static from '../../../src/ol/source/ImageStatic.js';
+import {get as getProjection, transform} from '../../../src/ol/proj.js';
+import ImageLayer from '../../../src/ol/layer/Image.js';
+
+const extent = [-123, 37, -122, 38];
+
+const source = new Static({
+  url: '/data/tiles/osm/5/5/12.png',
+  imageExtent: extent,
+  projection: getProjection('EPSG:4326')
```
Comment (ahocevar) [inline rendering/cases/reproj-image/main.js]: Acutally this should be the other way around, because the tiles are in EPSG:3857. You can configure the view to be in EPSG:4326 instead.

File: rendering/cases/reproj-image/main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Map from '../../../src/ol/Map.js';
+import View from '../../../src/ol/View.js';
+import Static from '../../../src/ol/source/ImageStatic.js';
+import {get as getProjection, transform} from '../../../src/ol/proj.js';
+import ImageLayer from '../../../src/ol/layer/Image.js';
+
+const extent = [-123, 37, -122, 38];
```
Comment (ahocevar) [inline rendering/cases/reproj-image/main.js]: Make sure that this extent matches the extent of the tile.

**Comment by KaiVolland** [comment]
Switched projections and cleaned up history.

**Comment by KaiVolland** [comment]
Oh i see. Thought the `tile.test.js` was just a copy of `image.test.js` but it contained way more tests. I'll see what i can do.

**Comment by ahocevar** [comment]
@KaiVolland In 789767d4119f8e25e284a1b4f8ba6e0d946bd0cf you can see exactly which tests were ported. I only deleted the portions from the old suite that are covered by the new test suite

**Comment by KaiVolland** [comment]
@ahocevar I transformed five more tests (EPSG:5070, ESRI:54009, date-line, northpole, none square) and force pushed.

**Comment by KaiVolland** [comment]
ping @ahocevar

**Comment by KaiVolland** [comment]
Any chance you can review this again @ahocevar? :kissing_cat:

**Comment by marcjansen** [comment]
I think this is fine to go. Anyone having objections?

**Comment by marcjansen** [comment]
Any objections for merging this @ahicevar? Otherwise I'll merge this. TIA
